.ci/Jenkinsfile

```diff
@@ -79,8 +80,13 @@ pipeline {
             withGithubNotify(context: 'Sanity checks', tab: 'tests') {
               deleteDir()
               unstash 'source'
-              dir("${BASE_DIR}"){
-                preCommit(commit: "${env.GIT_BASE_COMMIT}", junit: true)
+              script {
```

Comment:
Suddenly there was an environmental issue with the yamlint installation that's caused as part of the pre-commit. So I've decided to fix it as part of this PR.

Stacktrace:

```
[2021-01-06T13:54:56.246Z] + pre-commit install --install-hooks
[2021-01-06T13:54:56.508Z] pre-commit installed at .git/hooks/pre-commit
[2021-01-06T13:54:56.508Z] [INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[2021-01-06T13:54:57.087Z] [INFO] Initializing environment for https://github.com/detailyang/pre-commit-shell.git.
[2021-01-06T13:54:57.663Z] [INFO] Initializing environment for https://github.com/adrienverge/yamllint.git.
[2021-01-06T13:54:58.241Z] [INFO] Initializing environment for git@github.com:elastic/apm-pipeline-library.
[2021-01-06T13:54:59.189Z] [INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[2021-01-06T13:54:59.189Z] [INFO] Once installed this environment will be reused.
[2021-01-06T13:54:59.189Z] [INFO] This may take a few minutes...
[2021-01-06T13:55:07.358Z] [INFO] Installing environment for https://github.com/adrienverge/yamllint.git.
[2021-01-06T13:55:07.358Z] [INFO] Once installed this environment will be reused.
[2021-01-06T13:55:07.359Z] [INFO] This may take a few minutes...
[2021-01-06T13:55:08.304Z] An unexpected error has occurred: CalledProcessError: command: ('/var/lib/jenkins/workspace/apm-integration-tests_PR-1018/.cache/pre-commit/repoLT8dFM/py_env-python2.7/bin/python', u'/var/lib/jenkins/workspace/apm-integration-tests_PR-1018/.cache/pre-commit/repoLT8dFM/py_env-python2.7/bin/pip', 'install', '.')
[2021-01-06T13:55:08.304Z] return code: 1
[2021-01-06T13:55:08.304Z] expected return code: 0
[2021-01-06T13:55:08.304Z] stdout:
[2021-01-06T13:55:08.304Z]     Processing /var/lib/jenkins/workspace/apm-integration-tests_PR-1018/.cache/pre-commit/repoLT8dFM
[2021-01-06T13:55:08.304Z]     
[2021-01-06T13:55:08.304Z] stderr:
[2021-01-06T13:55:08.304Z]     DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
[2021-01-06T13:55:08.304Z]     ERROR: Package 'yamllint' requires a different Python: 2.7.17 not in '>=3.5.*'
[2021-01-06T13:55:08.304Z]     WARNING: You are using pip version 20.3.1; however, version 20.3.3 is available.
[2021-01-06T13:55:08.304Z]     You should consider upgrading via the '/var/lib/jenkins/workspace/apm-integration-tests_PR-1018/.cache/pre-commit/repoLT8dFM/py_env-python2.7/bin/python -m pip install --upgrade pip' command.
[2021-01-06T13:55:08.304Z]     
[2021-01-06T13:55:08.304Z] Check the log at /var/lib/jenkins/workspace/apm-integration-tests_PR-1018/.cache/pre-commit/pre-commit.log
```